spec/facility_spec.rb

```diff
@@ -2,25 +2,183 @@
 
 RSpec.describe Facility do
```

Comment:
Great job testing all of the functionality - I think you _could_ combine some of your `it` blocks related to registering a vehicle (only if you wanted to make these files more succinct :) )